### PR TITLE
Oai

### PR DESCRIPTION
**Pull request by davidgrove73**

### CODE REVIEW

File: app/controllers/adl_document_provider.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class AdlDocumentProvider < ::OAI::Provider::Base
+  repository_name 'Adl OAI provider'
+  repository_url  'http://localhost/provider'
```
Comment (ronan-mch) [inline app/controllers/adl_document_provider.rb]: configurable?

File: app/controllers/adl_document_provider.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  record_prefix ''
+  admin_email 'root@localhost'   # String or Array
```
Comment (ronan-mch) [inline app/controllers/adl_document_provider.rb:8]: as above... I think @xenma is adding a configurable admin email for her feedback form - should it be the same?

File: app/models/adl_document_wrapper.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+class AdlDocumentWrapper < ::OAI::Provider::Model
```
Comment (ronan-mch) [inline app/models/adl_document_wrapper.rb:8]: classdoc would be nice, even just a reference to other documentation if it's already covered elsewhere.

**Comment by ronan-mch** [comment]
Would it be worth adding a few line to the README about OAI so that Per or whoever it is has an idea of what's going on? Otherwise :+1:

**Comment by xenma** [comment]
:+1:

**Pull request merged**